lib/emojiController.js

```diff
@@ -0,0 +1,52 @@
+import { fabric } from 'fabric';
+
+class Emoji {
+  constructor(options = {}) {
+    this.options = options;
+    this.emojiAdded = false;
+    this.position = null;
+    this.options.fontSize = this.options.fontSize || 48;
+    const emoji = String.fromCodePoint(parseInt(this.options.iconCode.substr(2), 16));
+
+    if (!this.emojiAdded) {
+      this.text = new fabric.Text(emoji, {
```

Comment:
We could name it `this.emoji` so it might be easier in the future to know what it is :)